endpoints/get_all_objects.py

```diff
@@ -0,0 +1,17 @@
+import requests
+from endpoints.base_endpoint import Endpoint
+
+
+class GetAllObjects(Endpoint):
+    GET_ALL_ENTITIES = "/getAll"
+
+    def get_all_entities(self):
+        self.response = requests.get(f"{self.URL}{self.GET_ALL_ENTITIES}")
+        self.response_json = self.response.json().get('entity')
```

Comment:
Не нужно складывать какие-то данные в атрибут класса. Сделай как я описал выше, а уже внутри тестовой функции осуществляй проверки.
```suggestion
    def get_all_entities(self):
        return requests.get(f"{self.URL}{self.GET_ALL_ENTITIES}")
```